modules/calcite/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -164,6 +164,77 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     }
 }
 
+SqlNode IndexedColumn() :
+{
+    final Span s;
+    SqlNode col;
+}
+{
+    col = SimpleIdentifier()
+    (
+        <ASC>
+    |   <DESC> {
```

Comment:
do we have any plans to support NULL's ordering?

Reply:
Currently, it's not supported by the Ignite-H2 engine as well and not supported by core indexes. As a first goal, I think we should provide the same functionality as the Ignite-H2 engine, perhaps later NULL's ordering can be added. I can create another ticket if you think this functionality will be helpful.